anchors/ethereum_anchor_repository.go

```diff
@@ -37,12 +36,12 @@ type WatchAnchorCommitted interface {
 }
 
 type EthereumAnchorRepository struct {
-	config                   Config
+	config                   config.Config
```

Comment:
This breaks the pattern of what we are doing on other places. Can we just stick to one config interface per package for the sake of consistency? Using config.Config for mocking in tests is totally fine.
The reason for config per package is that its obvious and explicit what each package depends on rather than having a very broad definition of what it needs to function. Very useful for reuse. 

Excuse me as I didn't notice this change the previous review.

Reply:
Also go convention is to define the interface at the place of usage(anchors package) rather than the place of implementation(config package).